### PR TITLE
restructure question text

### PR DESCRIPTION
**Pull request by ClaraRosalie**
refs #73

### CODE REVIEW

**Comment by torsten-rathmann** [comment]
I think the author of these questions deliberately summarized them and had a case like this in mind: records often have to be split up to achieve a user-friendly download size. Having the questions in the Metadata section makes sense because the sharing products should definitely be linked via metadata. A help text can certainly help to clarify. I made a suggestion in Pull Request 90.

From my point of view, the question text does not need to be changed. But even if I cannot convince you, I will not prevent myself from emphasizing the individual questions. In this case, this pull request can still be combined with pull request 90 without any problems.

Unfortunately, I cannot deliver a French or Italian translation.

File: rdmorganiser/questions/rdmo.xml [hunk 1]
```diff
@@ -3640,25 +3640,33 @@ In caso affermativo, si prega di descrivere succintamente le convenzioni per i n
...
-		</text>
-		<text lang="it">Come sono strutturati i dati? In che modo sono collegati i componenti individuali gli uni agli altri?
-		Come sono collegati questi dati agli altri dati utilizzati nel progetto?
-		</text>
+		<text lang="fr">Veuillez expliquer brièvement la structure et l'interrelation de vos données:
+			
+* Quelle est la structure des données&nbsp;? 
+* Comment les composants individuels du jeu de données sont-ils liés les uns aux autres&nbsp;?
+* Comment le jeu de données est-il lié aux autres jeux de données utilisés dans le projet&nbsp;?</text>
+		<text lang="it">Spiega brevemente la struttura e l'interrelazione dei tuoi dati:
```
Comment (Zack-83) [inline rdmorganiser/questions/rdmo.xml:3665]:
```suggestion
		<text lang="it">Si prega di spiegare brevemente la struttura e l'interrelazione dei vostri dati:
```

**Review by Zack-83 — APPROVED** [review]

**Comment by Zack-83** [comment]
> 
> 
> I think the author of these questions deliberately summarized them and had a case like this in mind: records often have to be split up to achieve a user-friendly download size. Having the questions in the Metadata section makes sense because the sharing products should definitely be linked via metadata. A help text can certainly help to clarify. I made a suggestion in Pull Request 90.
> 
> From my point of view, the question text does not need to be changed. But even if I cannot convince you, I will not prevent myself from emphasizing the individual questions. In this case, this pull request can still be combined with pull request 90 without any problems.
> 
> Unfortunately, I cannot deliver a French or Italian translation.

I have no opinion - for me the question is fine both in the new and in the old version. I think the motivation for the change is described in Issue #73

**Review by jmfrenzel — APPROVED** [review]
Looks good. 
@torsten-rathmann do you aggree?

**Review by torsten-rathmann — APPROVED** [review]

**Pull request merged**